clusters/clusters_api_test.go

```diff
@@ -569,57 +571,6 @@ func TestPermanentDelete_Pinned(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestAccListClustersIntegration(t *testing.T) {
```

Comment:
why test is removed?

Reply:
It's moved

Reply:
moved to `clusters/acceptance/clusters_api_test.go`